adage/analyze/management/commands/import_data.py

```diff
@@ -12,6 +11,10 @@
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
 
+# Django imports
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
```

Comment:
I get why we want the django imports after the python libraries, but moving it below the code that sets up logging is going to make `flake8` cranky.

Oh, wait. `flake8` is already cranky about the import hacking below. Never mind. It makes sense to me to leave the logging setup above. Just know that once we correct the hack below we will still have a code quality bot complaining.